rdr_service/dao/consent_dao.py

```diff
@@ -171,13 +171,16 @@ def get_participants_needing_validation(cls, session) -> Collection[ParticipantS
         return query.all()
 
     @classmethod
-    def get_files_needing_correction(cls, session, min_modified_datetime: datetime = None) -> Collection[ConsentFile]:
-        query = session.query(ConsentFile).filter(
-            ConsentFile.sync_status == ConsentSyncStatus.NEEDS_CORRECTING
+    def get_next_revalidate_batch(cls, session, limit=1000) -> Collection[ConsentFile]:
+        query = (
+            session.query(ConsentFile.participant_id, ConsentFile.type)
```

Comment:
Nice, reducing the volume of the objects and attributes called should definitely help.